examples/structured_1d_dgsem/elixir_advection_shockcapturing.jl

```diff
@@ -42,8 +56,8 @@ function initial_condition_composite(x, t, equations::LinearScalarAdvectionEquat
 
     return SVector(value)
 end
-
-initial_condition = initial_condition_composite
+# Note calling the constructor of the struct: `InitialConditionComposite()` instead of `initial_condition_composite` !
+const initial_condition = InitialConditionComposite()
```

Comment:
Does setting this to `const` conflict with including different elixirs in a single REPL session?

Reply:
Yes, that will cause errors. I took inspiration from

https://github.com/trixi-framework/Trixi.jl/blob/f35d749f57c2d933941a39834f5833bfbdadb36a/src/equations/numerical_fluxes.jl#L244

Reply:
It is required there in our code but not here since we just pass the IC to the semidiscretization. So it should be fine without `const` here